README.rst

```diff
@@ -51,6 +51,20 @@ Add the ``health_check`` applications to your ``INSTALLED_APPS``:
         'health_check.contrib.s3boto_storage',      # requires boto and S3BotoStorage backend
     ]
 
+If you're using the celery health check, make sure your CELERY_QUEUES are updated (in settings.py).
```

Comment:
I think you mean `CELERY_TASK_ROUTES`, if in fact your settings prefix is `CELERY` which it doesn't have to be.

I would also recommend to add a link to the celery documentation here.
Also I believe it's `routing_key` not binding key, see http://docs.celeryproject.org/en/latest/userguide/routing.html#how-the-queues-are-defined

Reply:
If using CELERY_TASK_ROUTES, the current_app.amqp.queues will not see any queues. And setting.task_queues will not work in Django either (see https://github.com/celery/celery/pull/4467), so in the end I think this should stay as CELERY_QUEUES.
 Regarding the routing_key, yes here I think you're right - I've removed the dictionary settings and just referenced a link.

Reply:
If you resolve my [other comment](https://github.com/KristianOellegaard/django-health-check/pull/163#discussion_r160352231) it doesn't matter how you name your prefix. It will handle all. In this case I would just drop this line. Since there are many ways to configure multiple queues.